File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/gameframe/api/GameFrameApiInjector.java
package com.savvasdalkitsis.gameframe.injector.gameframe.api;

import com.savvasdalkitsis.gameframe.gameframe.api.GameFrameApi;

import static com.savvasdalkitsis.gameframe.injector.infra.network.RetrofitInjector.retrofit;

public class GameFrameApiInjector {

    public static GameFrameApi gameFrameApi() {
        return retrofit().create(GameFrameApi.class);
    }

}

File: app/src/main/java/com/savvasdalkitsis/gameframe/injector/gameframe/api/GameFrameApiInjector.kt
package com.savvasdalkitsis.gameframe.injector.gameframe.api

import com.savvasdalkitsis.gameframe.gameframe.api.GameFrameApi

import com.savvasdalkitsis.gameframe.injector.infra.network.RetrofitInjector.retrofit

object GameFrameApiInjector {

    fun gameFrameApi(): GameFrameApi = retrofit().create(GameFrameApi::class.java)

}
